dataPreProcessing.py

```diff
@@ -256,4 +256,32 @@ def remove_outliers(data):
 no_outliers_data = remove_outliers(data)
 
 # fig = px.box(data, y='Revenue')
-# fig.show()
\ No newline at end of file
+# fig.show()
+
+# Part 6 Normalization
+
+# Create a copy of the original DataFrame
+normalized_data_min_max = data.copy()
+normalized_data_mean_standardization = data.copy()
+
+# List of columns to normalize (numeric columns)
+numeric_columns = ['Inception', 'Employees', 'Revenue', 'Expenses', 'Profit', 'Growth']
+
+# Min-Max normalization for numeric columns
+min_vals = data[numeric_columns].min()
+max_vals = data[numeric_columns].max()
+normalized_data_min_max[numeric_columns] = (data[numeric_columns] - min_vals) / (max_vals - min_vals)
+
+# Mean-Standardization normalization for numeric columns
+mean_vals = data[numeric_columns].mean()
+std_vals = data[numeric_columns].std()
+normalized_data_mean_standardization[numeric_columns] = (data[numeric_columns] - mean_vals) / std_vals
```

Comment:
Not sure about the output of calculations. Range should be between 0 and 1 imo

Reply:
Found formulas in slides, please divide by root of dispersion.
<img width="949" alt="image" src="https://github.com/Ech0ke/data-science-machine-learning/assets/99315244/457be502-488e-4729-ad98-49cb142f66d4">